test/cli/dev.js

```diff
@@ -21,23 +22,37 @@ test('`fusion dev` works', async t => {
   t.end();
 });
 
-test('`fusion dev` works with assets', async t => {
+test.only('`fusion dev` works with assets', async t => {
```

Comment:
Remove .only

Reply:
good catch!

Reply:
(We should implement some sort of linter against this probably)